orca-core/src/main/groovy/com/netflix/spinnaker/orca/config/OrcaConfiguration.groovy

```diff
@@ -162,15 +165,11 @@ class OrcaConfiguration {
   }
 
   @Bean
-  ExecutionPropagationListener executionPropagationListenerBefore() {
-    // need a dedicated beforeJob listener due to how spring boot ordered listeners
-    new ExecutionPropagationListener(true, false)
-  }
-
-  @Bean
-  ExecutionPropagationListener executionPropagationListenerAfter() {
-    // need a dedicated afterJob listener due to how spring boot ordered listeners
-    new ExecutionPropagationListener(false, true)
+  CompositeExecutionListener executionListeners() {
```

Comment:
thoughts on just accepting an autowired List<ExecutionListener> 

moving these into explicit construction means that you can't DI a Listener

maybe even down below the places where a CompositeExecutionListener is expected, accept the list of listeners there and internally construct one of these